packages/cloud-cognitive/src/global/js/package-settings.js

```diff
@@ -13,10 +13,12 @@ const defaults = {
     // reviewed and released components:
     AboutModal: true,
     APIKeyModal: true,
+    Card: true,
```

Comment:
I don't think Card should be listed here -- it's not a component our end-users will use directly. It's internal, like TearsheetShell.

```suggestion
```

Reply:
This has been addressed in one of @davidmenendez's latest commits